packages/oats-runtime/src/make.ts

```diff
@@ -533,6 +444,113 @@ export function makeNullable(maker: any) {
   };
 }
 
+function isScalar(type: Type): boolean {
+  if (type.type === 'named') {
+    return isScalar(type.reference.definition);
+  }
+  return ['array', 'object', 'union', 'intersection', 'unknown'].indexOf(type.type) < -1;
+}
+
+function isEnum(type: Type) {
+  if (type.type === 'null') {
+    return true;
+  }
+  if (
+    type.type === 'string' ||
+    type.type === 'boolean' ||
+    type.type === 'number' ||
+    type.type === 'integer'
+  ) {
+    return !!type.enum;
+  }
+  return false;
+}
+
+function priority(v: ObjectType['properties'][0]) {
```

Comment:
I believe this is the intended way of getting the value from a type 🌝
```suggestion
function priority(v: ObjectType['properties'][string]) {
```